### PR TITLE
🏃enable <stylecheck> linter rule and fix related issues

### LINKED ISSUE #1538: cleanup: enable `stylecheck` linter
After #1535, we updated the golangci-lint to the latest version and matched the upstream Cluster API format for the linter, it seems there were a bunch of linters missing.

- Head over to [.golanci-lint.yaml](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/.golangci.yml) in the root of this repository.
- Remove the linter from the `linters.disable` list.
- Run `make lint-full`.
- Fix the reported errors with the suggestions.
- Submit a PR with the changes.

Feel free to reach out at any time if you need guidance or help.

/kind cleanup
/help
/good-first-issue
/milestone Next
/priority backlog

### PR DESCRIPTION
**Pull request by ernestas2k**
**What this PR does / why we need it**:
Enables stylecheck linter rule and fixes found violations.

**Which issue(s) this PR fixes**
Fixes #1538

### CODE REVIEW

File: cmd/clusterawsadm/cmd/alpha/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -207,15 +207,15 @@ func generateIAMPolicyDocJSON() *cobra.Command {
 				SharedConfigState: session.SharedConfigEnable,
 			})
 			if err != nil {
-				return fmt.Errorf("Error: failed to create a session: %v", err)
+				return fmt.Errorf("ERROR: failed to create a session: %v", err)
```
Comment (chuckha) [inline cmd/clusterawsadm/cmd/alpha/bootstrap/bootstrap.go]: Remove the `ERROR:` prefix. The convention in cluster-api is to return errors that are lower-cased and not prefixed.

Comment (ernestas2k) [reply cmd/clusterawsadm/cmd/alpha/bootstrap/bootstrap.go]: It's 5 occurrences of `fmt.Errorf("error: ....")` in this project, I'll fix all 5 of them in the same commit.

**Pull request merged**